### PR TITLE
CLOUD-91325 Image catalog V2 initial implementation

### PR DESCRIPTION
**Pull request by biharitomi**
@akanto @schfeca75 @keyki @mhmxs @topolyai5

### CODE REVIEW

File: cloud-common/src/main/resources/application-prod.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+cb.image.catalog.url: https://s3-eu-west-1.amazonaws.com/cloudbreak-info/v2-prod-cb-image-catalog.json
```
Comment (keyki) [inline cloud-common/src/main/resources/application-prod.yml]: Not public

File: cloud-common/src/main/resources/application-rc.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+cb.image.catalog.url: https://s3-eu-west-1.amazonaws.com/cloudbreak-info/v2-rc-cb-image-catalog.json
```
Comment (keyki) [inline cloud-common/src/main/resources/application-rc.yml]: Not public

File: cloud-common/src/main/resources/application-prod.yml (hunk 1 above)
Comment (biharitomi) [reply cloud-common/src/main/resources/application-prod.yml]: That will be available after the first release build.

File: cloud-common/src/main/resources/application-rc.yml (hunk 2 above)
Comment (biharitomi) [reply cloud-common/src/main/resources/application-rc.yml]: That will be available after the first rc build. But correct me @mhmxs , if I am wrong!

**Pull request merged**